tests/tests/test_basic_integration.py

```diff
@@ -171,14 +165,12 @@ def do_test_update_no_compression(
 
         # in the so-called clean image the artifact_info is "artifact_name=mender-image-clean\n"
         # see: https://github.com/mendersoftware/mender-qa/pull/580/commits/a4c54e9ebf5879f0837789cbc7c74cb1102b2caa#diff-7031fcd5c934f225781febe446a34594a9a36b335c540379c0dd376a8f06ba28R263
-        # so first we expect it to be set to master, as this is what we
-        # have in the initial image, after update we expect it to change
-        expected_artifact_info = f"artifact_name=mender-image-{mender_client_version}\n"
+        # so first we expect it to be different, after update we expect it to change to mender-image-clean
         artifact_info = mender_device.run(
             "cat /etc/mender/artifact_info | sed -e '/artifact_name/q'"
         )
         logger.info("before update got %s" % artifact_info)
-        assert artifact_info == expected_artifact_info
+        assert artifact_info != "artifact_name=mender-image-clean\n"
```

Comment:
maybe for clarity we should revert https://github.com/mendersoftware/integration/commit/5698c1628d0bba78930a0a48057a7035be553565 and apply the above fix? it will be only a couple of lines diff.